packages/code-studio/src/main/AppRouter.tsx

```diff
@@ -1,19 +1,13 @@
 import React, { ReactElement } from 'react';
-import {
-  BrowserRouter as Router,
-  Route,
-  Redirect,
-  Switch,
-} from 'react-router-dom';
+import { BrowserRouter as Router, Route, Switch } from 'react-router-dom';
 import AppInit from './AppInit';
 
 function AppRouter(): ReactElement {
   return (
     <Router basename={import.meta.env.BASE_URL}>
       <Switch>
-        <Route exact path="/" component={AppInit} />
         <Route path="/notebook/:notebookPath+" component={AppInit} />
-        <Redirect from="*" to="/" />
+        <Route path="*" component={AppInit} />
```

Comment:
Was the redirect causing an issue with the relative paths? It was to not show random URL paths that don't mean anything like `/ide/somethingIMadeUp`

Reply:
Yea this doesn't work correctly with the `'./'` at all, because it is just using the current URL path as the "basename".

I think instead, we should be getting the path from the `import.meta.url` (where are assets are). Not sure if the newer version of `react-router-dom` helps with this, but it has many breaking changes: https://reactrouter.com/en/main/upgrading/v5

For now I'll just fix up the base URL here.
Also it doesn't look like the `/notebook/` redirect works anywhere right now.

Reply:
Also, thinking about this even more - the `BASE_URL` set to `./` would screw up any routing we'd have with paths, since the index.html would be incorrect. e.g if I go to `/ide/notebook/foo.py`, it'll serve up the `index.html`, but the path to the assets will be incorrect, e.g.:
```
<script type="module" crossorigin src="./assets/index-05abad89.js"></script>
```
... Which would try and load `/ide/notebook/assets/index-05abad89.js`, which is of course incorrect.
Hmmm.